### ISSUE TITLE
How to set shortcut mod key to ctrl (ignoring left or right ctrl) ?

### ISSUE THREAD

**Issue by Mogez** [Genymobile/scrcpy#2102]
I've tried 
```
--shortcut-mod lctrl | rctrl
--shortcut-mod lctrl rctrl
--shortcut-mod lctrl + rctrl
```
so far without luck. Is this possible?

**Comment by rom1v** [comment]
```
scrcpy --shortcut-mod=lctrl,rctrl
```

https://github.com/Genymobile/scrcpy#shortcuts

**Comment by rom1v** [comment]
Note that if you use <kbd>Ctrl</kbd>, then you won't be able to use the Android shortcuts (injecting <kbd>Ctrl</kbd> to the device to copy-paste or trigger other Android shortcuts).

**Comment by Mogez** [comment]
I see it has to be a comma. Thanks.

Another thing, can ctrl+a (select all) be injected to Android?

**Comment by rom1v** [comment]
Yes, it is by default (if you don't use Ctrl for scrcpy shortcuts). That's the reason why the shortcut modifier has been changes to Alt by default, so that default Android shortcuts could work.
